01-Introduction/01-Introduction.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<ul><li>Double <code>and</code> in   <code>Specific science and and engineering domains.</code></li><li> <span class="ql-cursor">﻿</span></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done.</p>